tests/broker-engine/cma.robot

```diff
@@ -0,0 +1,361 @@
+*** Settings ***
+Documentation       Engine/Broker tests on centreon monitoring agent
+
+Library             Collections
+Library             DatabaseLibrary
+Library             DateTime
+Library             String
+
+Resource            ../resources/resources.resource
+
+Library             ../resources/Agent.py
+Library             ../resources/Broker.py
+Library             ../resources/Common.py
+Library             ../resources/Engine.py
+
+Suite Setup         Ctn Create Cert And Init
+Suite Teardown      Ctn Clean After Suite
+Test Setup          Ctn Stop Processes
+Test Teardown       Ctn Stop Engine Broker And Save Logs
+
+*** Test Cases ***
+
+BEOTEL_CENTREON_AGENT_CHECK_HOST
+    [Documentation]    agent check host and we expect to get it in check result
+    [Tags]    broker    engine    opentelemetry    MON-63843
+    Ctn Config Engine    ${1}    ${2}    ${2}
+    Ctn Add Otl ServerModule
+    ...    0
+    ...    {"otel_server":{"host": "0.0.0.0","port": 4317},"max_length_grpc_log":0, "centreon_agent":{"check_interval":10, "export_period":10}}
+    Ctn Config Add Otl Connector
+    ...    0
+    ...    OTEL connector
+    ...    opentelemetry --processor=centreon_agent --extractor=attributes --host_path=resource_metrics.resource.attributes.host.name --service_path=resource_metrics.resource.attributes.service.name
+    Ctn Engine Config Replace Value In Hosts    ${0}    host_1    check_command    otel_check_icmp
+    Ctn Set Hosts Passive  ${0}  host_1 
+
+    ${echo_command}   Ctn Echo Command   "OK - 127.0.0.1: rta 0,010ms, lost 0%|rta=0,010ms;200,000;500,000;0; pl=0%;40;80;; rtmax=0,035ms;;;; rtmin=0,003ms;;;;"
+
+    Ctn Engine Config Add Command    ${0}  otel_check_icmp   ${echo_command}    OTEL connector
+
+    Ctn Engine Config Set Value    0    log_level_checks    trace
+
+    Ctn Config Broker    central
+    Ctn Config Broker    module
+    Ctn Config Broker    rrd
+    Ctn Config Centreon Agent
+    Ctn Broker Config Log    central    sql    trace
+
+    Ctn Config BBDO3    1
+    Ctn Clear Retention
+
+    ${start}    Get Current Date
+    ${start_int}    Ctn Get Round Current Date
+    Ctn Start Broker
+    Ctn Start Engine
+    Ctn Start Agent
+
+    # Let's wait for the otel server start
+    ${content}    Create List    unencrypted server listening on 0.0.0.0:4317
+    ${result}    Ctn Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    10
+    Should Be True    ${result}    "unencrypted server listening on 0.0.0.0:4317" should be available.
+    Sleep    1
+
+    ${result}    Ctn Check Host Check Status With Timeout    host_1    30    ${start_int}    0    OK - 127.0.0.1
+    Should Be True    ${result}    hosts table not updated
```

Comment:
I believe we also have a function to check status in resources. It seems more relevant as we should all use it now.